src/FM.LiveSwitch.Mux.Standard/FileSizeMeasurement.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+
+namespace FM.LiveSwitch.Mux
+{
+    public class FileSizeMeasurement
```

Comment:
If this class isn't serialized anywhere, we don't need the Newtonsoft.Json dependency.